catalog/OpenET/OpenET_GEESEBAL_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet

```diff
@@ -0,0 +1,164 @@
+    USDA Cropland Data Layer (CDL) and filters for NDVI, LST and albedo; (ii) corrections to LST for
+    endmembers based on antecedent precipitation; (iii) definition of NLDAS wind speed thresholds to reduce
+    model instability during the atmospheric correction; and, (iv) improvements to estimate daily net
+    radiation, using FAO-56 as reference (Allen et al., 1998). Overall, geeSEBAL performance is dependent on
+    topographic, climate, and meteorological conditions, with higher sensitivity and uncertainty related to hot
+    and cold endmember selections for the CIMEC automated calibration, and lower sensitivity and uncertainty
+    related to meteorological inputs (Laipelt et al., 2021 and Kayser et al., 2022). To reduce uncertainties
+    related to complex terrain, improvements were added to correct LST and global (incident) radiation on the
+    surface (including the environmental lapse rate, elevation slope and aspect) to represent the effects of
+    topographic features on the model’s endmember selection algorithm and ET estimates.
+
+    [Additional information](https://openetdata.org/methodologies/)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'conus',
+    'evapotranspiration',
+    'gridmet_derived',
+    'landsat_derived',
+    'monthly',
+    'water',
+    'geesebal'
+  ],
+  providers: [
+    ee.producer_provider('OpenET, Inc.', 'https://openetdata.org/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-126.0, 25.0, -66.0, 50.0, '2016-01-01T00:00:00Z', null),
+  summaries: {
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'et',
+        description: |||
+          geeSEBAL ET value
+        |||,
+        'gee:units': 'mm',
+      },
+
+      {
+        name: 'count',
+        description: |||
+           Number of cloud free values
+        |||,
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'OpenET geeSEBAL Monthly ET',
+        lookat: {
+          lat: 38.0,
+          lon: -100.0,
+          zoom: 5,
+        },
+        image_visualization: {
+          band_vis: {
+            min: [
+              0,
+            ],
+            max: [
+              1400.0,
+            ],
+            palette: [
+              '9E6212',
+              'AC7D1D',
+              'BA9829',
+              'C8B434',
+              'D6CF40',
+              'BED44B',
+              '9FCB51',
+              '80C256',
+              '61B95C',
+              '42B062',
+              '45B677',
+              '49BC8D',
+              '4DC2A2',
+              '51C8B8',
+              '55CECE',
+              '4DB4BA',
+              '459AA7',
+              '3D8094',
+              '356681',
+              '2D4C6E',
+            ],
+            bands: [
+              'et',
+            ],
+          },
+        },
+      },
+    ],
+  },
+  'sci:doi': '10.1111/1752-1688.12956',
+  'sci:citation': |||
```

Comment:
Just one citation for the data.  You can then have multiple sci:publications.  For example, see here:

https://github.com/google/earthengine-catalog/blob/main/catalog/NOAA/NOAA_GOES_17_MCMIPC.jsonnet#L1029

Reply:
As above, please add doi links.